binaries/upload.py

```diff
@@ -23,21 +23,29 @@
 PACKAGES = ["torchserve", "model-archiver", "workflow-archiver"]
 
 
-def upload_pypi_packages():
+def upload_pypi_packages(test_pypi=False):
     """
     Takes a list of path values and uploads them to pypi using twine, using token stored in environment variable
     """
     os.system(f"pip3 install twine -q")
```

Comment:
Let's remove this, `twine` should be included in our dev dependencies

Reply:
Done